src/main.py

```diff
@@ -1,6 +1,29 @@
-from cv2 import cv2
+"""Main file holding code responsible for running the program."""
+from threading import Thread
+import os
+import pickle
+import cv2
 from detectEyes import detectEyes
 from ui.runners import launchUIThread
+from ui.widgets import CALIBRATION_FILE_NAME
+
+
+class IrisSoftware:
+    """Main class responsible for running the program."""
+
+    def __init__(self) -> None:
+        # State
+        self.isCalibrated = False
+
+    def run(self) -> None:
+        # Check for calibration data
+        if os.path.exists(CALIBRATION_FILE_NAME):
```

Comment:
As written, this would check for the existence of a calibration file once when the program first runs, but then UI has to run for the user to calibrate it before the calibration file can exist, so we should probabally add a check to the main while loop of our program so it constantly checks, or equivalently, check for messages from the UI thread and recalibrate whenever the UI thread tells it that the user ran the calibration screen.

Reply:
Yeah agreed - my idea here was just to future-proof ourselves. If we have calibration data, then we can safely start up our eye to screen mapping. If not though, then we should ask the user to calibrate the program first before starting up the eye to screen mapping. Was planning on addressing this in a different PR though since this one is getting large.